.golangci.yml

```diff
@@ -82,6 +82,10 @@ issues:
         - forbidigo
       # this is the one file we want to allow exec.Cmd functions in
       path: private/pkg/command/runner.go
+    - linters:
+        - forbidigo
+        # we use os.Rename here to rename files in the same directory
+      path: private/bufpkg/bufmodule/bufmodulecache/cas_module_cacher.go
```

Comment:
I looked at when this rule was added and it was when we attempted to rename files across filesystems. In this case, I'm renaming a file within the same directory where it will ultimately exist, so I think we should relax this check.

Reply:
Nit: you can regex the exact string match, to avoid excluding the whole file.